src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "variables";
+
+form, .custom-code {
+    width: 400px;
```

Comment:
Should we set this 400px as a css variable? I think we want a lot of the forms in the system to be this width, no?